docs/modules/ROOT/images/advanced_configuration/webui/topbar/setup_topbar.py

```
@@ -0,0 +1,58 @@
+            diagnostics = d.find_element(
+                By.XPATH,
+                '//nav[contains(@class, "topbar")]//a[contains(normalize-space(), "Diagnostics")]',
+            )
+            logs = d.find_element(
+                By.XPATH,
+                '//nav[contains(@class, "topbar")]//a[contains(normalize-space(), "Logs")]',
+            )
+            ping_host = d.find_element(By.CSS_SELECTOR, '[title="Ping host"]')
+            restart = d.find_element(By.CSS_SELECTOR, '[title="Restart Docker Container"]')
+            delete_backups = d.find_element(By.CSS_SELECTOR, '[title="Delete old backups"]')
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)